### PR TITLE
BUD-1116 Api documentation changes for budget api

### PR DESCRIPTION
**Pull request by anuvenkataraman**
## Pull Request Checklist

**Complete and check all items that are applicable.**

- [ ] Proofread documentation changes for spelling and grammatical errors
- [ ] Used Markdown code blocks for all applicable examples using code
- [ ] Used relative links when linking to other documentation on Developer Center
- [ ] If creating new API reference pages, followed the guidelines stated [in the wiki](https://github.com/concur/developer.concur.com/wiki/Creating-API-Reference-pages)
- [ ] If moving pages, added [page redirects](https://github.com/concur/developer.concur.com/wiki/Adding-Redirects) so users are redirected to the new page

### CODE REVIEW

File: src/api-reference/budget/budget-category.md [hunk 1]
```diff
@@ -3,18 +3,24 @@ title: Budget Category
...
 * [Retrieve all Budget Categories](#getall)
 * [Retrieve a Budget Category](#get)
 * [Create/Update a Budget Category](#post)
 * [Remove a Budget Category](#delete)
 * [Retrieve all valid Expense Types](#getExpTypes)
 * [Schema](#schema)
 
+## Overview
+
+The new Budget Service API is in Beta. If you are interested in using the Budget Service API, then please contact you account manager for further details.
```
Comment (jtbigwoo) [inline src/api-reference/budget/budget-category.md]: should be "contact your account manager"

File: src/api-reference/budget/budget-header.md [hunk 2]
```diff
@@ -3,16 +3,21 @@ title: Budget Header
...
+* [Overview](#overview)
 * [Retrieve all Budget Item Headers](#getall)
 * [Retrieve a Budget Item Header](#get)
 * [Create/Update a Budget Item Header](#post)
 * [Remove a Budget Item Header](#delete)
 * [Schema](#schema)
 
+### Overview
+
+The new Budget Service API is in Beta. If you are interested in using the Budget Service API, then please contact you account manager for further details.
```
Comment (jtbigwoo) [inline src/api-reference/budget/budget-header.md]: should be "contact your account manager"

File: src/api-reference/budget/fiscal-year.md [hunk 3]
```diff
@@ -5,12 +5,16 @@ layout: reference
...
+* [Overview](#overview)
 * [Retrieve all Fiscal Years](#getall)
 * [Retrieve a Fiscal Year](#get)
 * [Create/Update a Fiscal Year](#post)
 * [Remove a Fiscal Year](#delete)
 * [Schema](#schema)
 
+### Overview
+
+The new Budget Service API is in Beta. If you are interested in using the Budget Service API, then please contact you account manager for further details.
```
Comment (jtbigwoo) [inline src/api-reference/budget/fiscal-year.md]: should be "contact your account manager"

File: src/api-reference/budget/getting-started.md [hunk 4]
```diff
@@ -13,9 +13,12 @@ layout: reference
 - [Explore the API](#explore-the-api)
   - [Prerequisites](#prerequisites)
   - [Acquire an Access Token](#retrieve-a-user-access-token)
+  - [Roles & Scopes](#roles-and-scopes)
 
 ### Overview
 
+The new Budget Service API is in Beta. If you are interested in using the Budget Service API, then please contact you account manager for further details.
```
Comment (jtbigwoo) [inline src/api-reference/budget/getting-started.md]: should be "contact your account manager"

**Comment by charlieconcur** [comment]
We've got both `cUrl` and `HTTPie` command line examples which is (a) a duplication and (b) shows a preference for tools (leaving out others like Postman an Insomnia). The current guidance is to make this generic as shown in https://github.concur.com/developer/api/blob/master/documentation-template.md. Here is the excerpt + example using one of this APIs examples.

**Template**

```
{HTTP Method} {URL}
{Header}: {Value}
```

**Example**

```
POST https://us.api.concursolutions.com/budget/v4/budgetCategory
Authorization: Bearer {access-token}
Content-Type: application/json
```

Let's take the time to resolve both (a) and (b) by making the information generic -- developers can easily translate these into their tooling of choice.

**Comment by charlieconcur** [comment]
@anuvenkataraman ping.

**Comment by anuvenkataraman** [comment]
closing this for now as we need more changes

**Pull request closed without merging**